concepts/Common/Integration Guides/70 Manage Azure Blob Storage with DevExtreme Components/40 FileUploader.md

```diff
@@ -0,0 +1,55 @@
+
+The `FileUploader` component can upload blobs to Azure Storage with the multi-part upload technique.
```

Comment:
Is the word "technique" correct, or are you talking about a "feature"?

Reply:
Technique is correct here — we're talking about an abstract approach to file uploading, not a particular capability